osquery/logger/logger.cpp

```diff
@@ -217,9 +208,6 @@ class BufferedLogSink : public google::LogSink, private boost::noncopyable {
 
   /// Mutex for checking primary status.
   mutable Mutex primary_mutex_;
-
-  /// Mutex protecting activation and enabling of the buffered status logger.
-  Mutex enable_mutex_;
```

Comment:
no need for active_ anymore. This mutex was protecting active.

Reply:
its weird but yeah, enabled_ is atomic. active_ is not.

Reply:
Also, if someone is in doubt, RemoveLogSink and AddLogSink are protected by their own lock.